src/main/java/teammates/ui/controller/InstructorEditInstructorFeedbackSaveAction.java

```diff
@@ -0,0 +1,205 @@
+package teammates.ui.controller;
+
+import teammates.common.datatransfer.FeedbackParticipantType;
+import teammates.common.datatransfer.FeedbackQuestionAttributes;
+import teammates.common.datatransfer.FeedbackSessionAttributes;
+import teammates.common.datatransfer.FeedbackSessionQuestionsBundle;
+import teammates.common.datatransfer.InstructorAttributes;
+import teammates.common.exception.EntityDoesNotExistException;
+import teammates.common.exception.InvalidParametersException;
+import teammates.common.exception.UnauthorizedAccessException;
+import teammates.common.util.Assumption;
+import teammates.common.util.Const;
+import teammates.common.util.HttpRequestHelper;
+import teammates.common.util.StatusMessage;
+import teammates.common.util.Const.StatusMessageColor;
+import teammates.logic.api.GateKeeper;
+
+/**
+ * The {@code InstructorEditInstructorFeedbackSaveAction} class handles incoming requests to 
+ * save the data after moderating the instructor.
+ */
+public class InstructorEditInstructorFeedbackSaveAction extends FeedbackSubmissionEditSaveAction {
+    InstructorAttributes moderatedInstructor;
+    
+    /**
+     * Verifies if the user is allowed to carry out the action.
+     */
+    @Override
+    protected void verifyAccesibleForSpecificUser() {
+        InstructorAttributes instructor = logic.getInstructorForGoogleId(courseId, account.googleId);
+        FeedbackSessionAttributes session = logic.getFeedbackSession(feedbackSessionName, courseId);
+
+        new GateKeeper().verifyAccessible(instructor,
+                session,
+                false, 
+                Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_SESSION);
+    }
+    
+    /**
+     * Retrieves any additional parameters from request and set them accordingly.
+     */
+    @Override
+    protected void setAdditionalParameters() throws EntityDoesNotExistException {
+        String moderatedInstructorEmail = getRequestParamValue(Const.ParamsNames.FEEDBACK_SESSION_MODERATED_PERSON);
+        Assumption.assertPostParamNotNull(Const.ParamsNames.FEEDBACK_SESSION_MODERATED_PERSON, moderatedInstructorEmail);
+
+        moderatedInstructor = logic.getInstructorForEmail(courseId, moderatedInstructorEmail);
+        
+        // If the instructor doesn't exist
+        if (moderatedInstructor == null) {
```

Comment:
Yeah. I realised why I added it. Because it was suggested to add it @kanghj 
![screen shot 2016-04-10 at 7 10 43 pm](https://cloud.githubusercontent.com/assets/10682388/14409805/2db3eb10-ff50-11e5-8f69-1a869a861d3c.png)